websites/D/DramaCool/presence.ts

```diff
@@ -74,8 +71,13 @@ presence.on("UpdateData", async () => {
 		presenceData.smallImageKey = "search";
 		presenceData.smallImageText = strings.searching;
 	} else if (pathname.match("/([a-z0-9-]+)-episode-([0-9]+)")) {
-		ShowData.title = document.querySelector("div.category a").textContent;
-
+		ShowData.title =
+			document.querySelector("div.category a")?.textContent ??
+			JSON.parse(
+				document
+					.querySelector('[class="yoast-schema-graph"]')
+					.innerHTML.replace(/@/gm, "")
+			).graph[3].itemListElement[1].name.replace(/Episode [0-9]*/gm, "");
```

Comment:
And now this should not be necessary

Reply:
Easiest way to get the title that doesn't change.